components/brave_wallet/browser/brave_wallet_utils.cc

```diff
@@ -706,11 +795,12 @@ GURL GetNetworkURL(PrefService* prefs,
                    const std::string& chain_id,
                    mojom::CoinType coin) {
   if (coin == mojom::CoinType::ETH) {
-    mojom::NetworkInfoPtr known_network = GetKnownEthChain(prefs, chain_id);
-    if (!known_network)
-      return GetCustomChainURL(prefs, chain_id);
+    GURL custom_chain_url = GetCustomChainURL(prefs, chain_id);
```

Comment:
I think it's safer to still return a known chain here first rather than switch to a custom being returned first because the known chainURL's we're vetting to make sure they're trusted where as the custom ones are relying on the user to vett them.